pkg/cmd/pipeline/list.go

```diff
@@ -21,6 +21,8 @@ import (
 	"text/tabwriter"
 	"text/template"
 
+	"github.com/tektoncd/cli/pkg/helper/pipeline"
```

Comment:
needs to be `goimported` (aka no extra space :hugs: )

Reply:
My IDE ...